app/models/concerns/mergeable.rb

```diff
@@ -82,12 +86,16 @@ def update_existing_record(existing_record, attributes)
       existing_record
     end
 
-    def return_old_record(existing_record, new_record)
+    def return_identical_record(existing_record)
+      logger.debug "#{self} with id #{existing_record.id} is identical, doing nothing."
+      increment_metric(:identical)
+      existing_record.merge_status = :identical
```

Comment:
Do we need to account for this new merge_status [over here](https://github.com/simpledotorg/simple-server/blob/865f3b4324b8941687ee823190a1d968586da0e3/app/services/merge_patient_service.rb#L35)?  I'm not sure if these paths ever intersect, but I imagine they _could_ sometime in the future, and we would probably want to ignore `identical` in the MergePatientService.

Reply:
I took a look, and the current decision tree is still accurate. It's only concerned with whether or not to set the registration user and facility from the request, or keep the existing info. If the record's `identical`, it's an existing record, so it should keep its existing regn info. The `else` clause does that correctly. After that, the service invokes `Patient.merge`, which now is a no-op as of this PR.